lib/hpcbase.pm

```diff
@@ -24,6 +24,12 @@ sub exec_and_insert_password {
     send_key "ret";
 }
 
+sub enable_and_start {
```

Comment:
1. enable_and_start_service ?
2. I think it worth to be added to utils.pm it is not HPC specific